modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -373,6 +373,19 @@ Extra Field Set:: Specifies extra fields.
 Registrations Require Approval:: Select whether registration requires approval.
 Approver ID:: Specifies the approver ID.
 
+== Spec Downloader
+
+The Spec Downloader component enables users to download an API specification.
+
+The API specification downloaded from API Community Manager is the same as the specification downloaded from Anypoint Exchange.
+
+The available buttons are *Download Spec as RAML* and *Download Spec as OAS*. If the API specification is uploaded as RAML and downloaded as OAS, or uploaded as OAS and downloaded as RAML, then the downloaded version is automatically generated and may have small inconsistencies with the uploaded version.
+
+You can specify values for the following parameters of the Spec Downloader component:
+
+Record Id:: Specifies the API. Default is `{!recordId}`, which downloads the specification of the API selected on the current page.
+Layout:: Specifies the layout of the buttons, `Horizontal` or `Vertical`. Default is `Horizontal`.
+
 == Filter Conditions [[filter-conditions]]
```

Comment:
```suggestion
== Filter Conditions Parameter [[filter-conditions]]
```

Reply:
This one is also an exact component name, so I'm preserving the current text.